packages/core/src/Cardano/types/StakePool/StakePool.ts

```diff
@@ -53,40 +107,64 @@ export enum StakePoolStatus {
 }
 
 /**
- * Stake pool performance per epoch, taken at epoch rollover
+ * Stake pool performance per epoch, taken at epoch rollover.
  */
 export class StakePoolEpochRewards {
   /**
-   * Epoch length in milliseconds
+   * Epoch length in milliseconds.
```

Comment:
Is this still milliseconds :thinking:?

Reply:
I am also not sure about this, should be measured in slots rather than ms. I will look in to the query to make sure.

Reply:
Slots make more sense to me. Slot length is in measured in time.

Reply:
We could convert it to slots, but not sure if this is needed as ms for some calculation, maybe we should renamed it in that case

Reply:
Yeah it probably is used. Maybe let's just leave it as-is for now.